src/utils/documentUtils.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import JSZip from "jszip";
+import { docPropsCoreXmlPath, docPropsRootElement } from "../constants";
+
+export const createOrUpdateProperty = (
+    doc: Document,
+    parent: Element,
+    property: string,
+    value: string | null | undefined
+): void => {
+    if (!value) {
```

Comment:
this `if(!value)` check catches both the case for null and undefined.
My intent was to have null clear out the value if it exists, and undefined to be no-op for a specific property.